pyproject.toml

```diff
@@ -36,8 +36,18 @@ quote-style = "single"
 indent-style = "space"
 
 [tool.ruff.lint]
-select = ["E", "F", "B"]
-ignore = ["E501"]
+select = [
+  "E", # pycodestyle errors
+  "F", # pyflakes
+  "B", # bugbear (likely bugs)
+  "I", # sort imports
+  "RUF022", # sort __all__
+]
+ignore = [
+  "E111", # indentation not a multiple of four
+  "E114", # indentation not a multiple of four
```

Comment:
were these giving issues?

Reply:
When running with `--preview`, which I momentarily needed for something, yeah, but this is a stray change. Removed!